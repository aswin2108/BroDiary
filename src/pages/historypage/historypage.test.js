import renderer from 'react-test-renderer';
import HistoryPage from './historyPage.component';

it("History page rendered properly", ()=>{
    const historyPage=renderer.create(
        <HistoryPage/>
    ).toJSON();
    expect(historyPage).toMatchSnapshot();
});