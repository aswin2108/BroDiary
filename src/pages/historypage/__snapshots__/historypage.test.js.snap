// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History page rendered properly 1`] = `
<div
  style={
    Object {
      "left": "50%",
      "position": "fixed",
      "top": "50%",
      "transform": "translate(-50%, -50%)",
    }
  }
>
  <span
    style={
      Object {
        "display": "inherit",
        "height": "100px",
        "position": "relative",
        "width": "100px",
      }
    }
  >
    <span
      style={
        Object {
          "animation": "react-spinners-CircleLoader-circle 1s 0s infinite linear",
          "animationFillMode": "",
          "borderBottom": "none",
          "borderLeft": "1px solid #000000",
          "borderRadius": "100%",
          "borderRight": "none",
          "borderTop": "1px solid #000000",
          "height": "100px",
          "left": "0%",
          "position": "absolute",
          "top": "0%",
          "transition": "2s",
          "width": "100px",
        }
      }
    />
    <span
      style={
        Object {
          "animation": "react-spinners-CircleLoader-circle 1s 0.2s infinite linear",
          "animationFillMode": "",
          "borderBottom": "none",
          "borderLeft": "1px solid #000000",
          "borderRadius": "100%",
          "borderRight": "none",
          "borderTop": "1px solid #000000",
          "height": "90px",
          "left": "0.875%",
          "position": "absolute",
          "top": "1.75%",
          "transition": "2s",
          "width": "90px",
        }
      }
    />
    <span
      style={
        Object {
          "animation": "react-spinners-CircleLoader-circle 1s 0.4s infinite linear",
          "animationFillMode": "",
          "borderBottom": "none",
          "borderLeft": "1px solid #000000",
          "borderRadius": "100%",
          "borderRight": "none",
          "borderTop": "1px solid #000000",
          "height": "80px",
          "left": "1.75%",
          "position": "absolute",
          "top": "3.5%",
          "transition": "2s",
          "width": "80px",
        }
      }
    />
    <span
      style={
        Object {
          "animation": "react-spinners-CircleLoader-circle 1s 0.6000000000000001s infinite linear",
          "animationFillMode": "",
          "borderBottom": "none",
          "borderLeft": "1px solid #000000",
          "borderRadius": "100%",
          "borderRight": "none",
          "borderTop": "1px solid #000000",
          "height": "70px",
          "left": "2.6249999999999996%",
          "position": "absolute",
          "top": "5.249999999999999%",
          "transition": "2s",
          "width": "70px",
        }
      }
    />
    <span
      style={
        Object {
          "animation": "react-spinners-CircleLoader-circle 1s 0.8s infinite linear",
          "animationFillMode": "",
          "borderBottom": "none",
          "borderLeft": "1px solid #000000",
          "borderRadius": "100%",
          "borderRight": "none",
          "borderTop": "1px solid #000000",
          "height": "60px",
          "left": "3.5%",
          "position": "absolute",
          "top": "7%",
          "transition": "2s",
          "width": "60px",
        }
      }
    />
  </span>
</div>
`;
